lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -455,29 +441,21 @@ class MouseAdapter extends BaseAdapter {
     ui.PointerChange change,
     html.MouseEvent event,
   ) {
-    final List<ui.PointerData> data = <ui.PointerData>[];
-    // The mousedown event will cause an 'add' event on the framework side.
-    if (event.type == 'mousedown') {
-      PointerBinding._instance._activePointerIds.add(_mouseDeviceId);
-    }
-    if (event.type == 'mousemove') {
```

Comment:
If we receive move without pointer is previously added, do we consider this as a valid transition in flutter web? @yjbanov 
This will previously through exception in framework because we will also need to synthesize and down to make it work. I added that logic in this pr anyway, but was wondering if that is necessary.
https://github.com/chunhtai/engine/blob/cefc3820a3089617039f787685cee2e871d2a66f/lib/web_ui/lib/src/engine/pointer_converter.dart#L186

Reply:
I can see how having "down" is essential for touch events, but what does it mean for a mouse? Do you always have to press a mouse button before you start moving it? That would always produce a drag. Mouse can just hover over things and never send down/up events.

/cc @mdebbar

Reply:
@yjbanov there is a separate `PointerChange.hover` for the case you are describing.

Reply:
This makes more sense now. For some reason, i keep thinking mousemove directly translate to PointerData.change=move and forgetting it could also means a hover.